### PR TITLE
{lib}[system/system] libtool v2.4.7

### PR DESCRIPTION
**Pull request by dithwick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libtool/libtool-2.4.7.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+"""
+
+toolchain = SYSTEM
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['04e96c2404ea70c590c546eba4202a4e12722c640016c12b9b2f1ce3d481e9a8']
+
+dependencies = [
+    ('M4', '1.4.18'),
```
Comment (smoors) [inline easybuild/easyconfigs/l/libtool/libtool-2.4.7.eb:20]: @dithwick why not use the more recent version `1.4.19` here?

Comment (dithwick) [reply easybuild/easyconfigs/l/libtool/libtool-2.4.7.eb:20]: Hi @smoors.

I was going to originally but then in the process of putting together the system toolchain version of the Autotools-20220317 release, I noticed Autoconf-2.71.eb uses the 1.4.18 release of M4 so figured if I made it the 1.4.18 release here, only one version of M4 needed to be built. Happy to change it to 1.4.19, or should I do an update to Autoconf-2.71.eb so it uses 1.4.19 instead?

Comment (smoors) [reply easybuild/easyconfigs/l/libtool/libtool-2.4.7.eb:20]: no it's ok, thanks.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node301.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/897b9e08025852f96a410e63470f1187 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @dithwick!

**Pull request merged**